patches/chrome-browser-profiles-profile_impl.cc.patch

```diff
@@ -10,13 +10,3 @@ index 71619ca3d5271b0f813110dce0e5570b6eebdda7..a9c1f8ec99aafa581c7fc8261e470261
  #include "build/build_config.h"
  #include "chrome/browser/background/background_contents_service_factory.h"
  #include "chrome/browser/background_fetch/background_fetch_delegate_factory.h"
-@@ -508,6 +509,9 @@ ProfileImpl::ProfileImpl(
-   LoadPrefsForNormalStartup(async_prefs);
- #endif
- 
-+// Done here instead of browser_prefs.cc so it can override default prefs
```

Comment:
@darkdh did you verify that this still works correctly?

Reply:
yes, if we override default at wrong place, we will get a DCHECK failure saying using unregistered prefs